Rakefile

```
@@ -17,3 +17,18 @@ task :test do
 end
 
 task default: :test
+
+task :get_xkcd do
+  xkcd = GetColors::XKCD.new
+  xkcd.get_data
+  xkcd.dump_file("lib/colors/color_data/xkcd.rb")
+end
+
+task :get_css4 do
+  css4 = GetColors::CSS4.new
+  css4.get_data
+  css4.dump_file("lib/colors/color_data/css4.rb")
+end
+
+desc "Get color data files from the web, per issue #8"
```

We don't need to refer #8 here.

```suggestion
desc "Get color data files from the web"
```